libraries/AP_UAVCAN/AP_UAVCAN.cpp

```diff
@@ -84,6 +86,19 @@ const AP_Param::GroupInfo AP_UAVCAN::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("SRV_RT", 4, AP_UAVCAN, _servo_rate_hz, 50),
 
+    // @Param: IMUD_M
+    // @DisplayName: IMU output index
+    // @Description: Bitmask describing which IMU deltas to publish
+    // @Bitmask: 0: IMU 1, 1: IMU 2, 2: IMU 3
+    // @User: Advanced
+    AP_GROUPINFO("IMUD_M", 5, AP_UAVCAN, _imu_pub_mask, 0),
+
+    // @Param: IMUD_F
```

Comment:
I'd prefer a _RT suffix, to match the existing SRV_RT param in the same structure

Reply:
Again, I don't believe it will fit.